### PR TITLE
Fixed bug. Added in check for ufunc and evaluates inner expression be…

### LINKED ISSUE #24670: 'BinOp' object has no attribute 'value' Issue while calling global functions in eval and importing numpy
```python
import pandas as pd
from  numpy import *
pd.eval("floor(1)")
Output: 1.0
pd.eval("floor(1+2)")
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/eval.py", line 291, in eval
    truediv=truediv)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 739, in __init__
    self.terms = self.parse()
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 756, in parse
    return self._visitor.visit(self.expr)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 321, in visit
    return visitor(node, **kwargs)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 327, in visit_Module
    return self.visit(expr, **kwargs)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 321, in visit
    return visitor(node, **kwargs)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 330, in visit_Expr
    return self.visit(node.value, **kwargs)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 321, in visit
    return visitor(node, **kwargs)
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 579, in visit_Call_35
    new_args = [self.visit(arg).value for arg in node.args]
  File "/Users/sanjana/simulation-service/venv/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 579, in <listcomp>
    new_args = [self.visit(arg).value for arg in node.args]
AttributeError: 'BinOp' object has no attribute 'value'

### PR DESCRIPTION
**Pull request by SurajH1**
…fore evaluating outer expression

- [x] closes  #24670 
- [x] passes `black pandas`

### CODE REVIEW

File: pandas/core/computation/expr.py [hunk 1]
```diff
@@ -815,3 +822,5 @@ def names(self):
 
 
 _parsers = {"python": PythonExprVisitor, "pandas": PandasExprVisitor}
+
+
```
Comment (jbrockmendel) [inline pandas/core/computation/expr.py:826]: this is going to cause the lint check in the CI to fail

Comment (SurajH1) [reply pandas/core/computation/expr.py]: resolved this issue

**Comment by SurajH1** [comment]
Is this ready to be accepted @maintainters

**Review by jreback — CHANGES_REQUESTED** [review]
pls write tests first

**Comment by SurajH1** [comment]
any suggestions on why the failures are only occurring on some systems? @jreback

**Comment by vmuriart** [comment]
I tested this branch locally using a function that makes multiple arguments, and it isn't working properly either. It isn't parsing the multiple arguments correctly. 

```python
import pandas as pd
from  numpy import *

print(pd.__version__)

r = pd.eval("floor(1+5)")
print(r)

r = pd.eval("minimum(1 + 5, 4)")  # This one fails
print(r)
```

**Comment by vmuriart** [comment]
> any suggestions on why the failures are only occurring on some systems? @jreback

Looking at the Azure CI output logs, it looks like you are failing on the environments that are testing the minimum versions, in particular, the lowest version of `numexpr`. There seems to be an `if` branch that specifically tries to handle the `floor` function. You could try written the test with a different function that isn't `floor`, or fix the if statement causing this error. (`pandas/core/computation/ops.py:602: ValueError`)

The Travis CI failures look like they are on environments that have `numba` installed. Here I am totally guessing, but I imagine that the failing operations use the `eval` capability somewhere internally, and that your changes are causing them to now fail.

**Comment by SurajH1** [comment]
Do you think the same error would occur if we changed it from a check for `floor ` to a check for `np.ufunc`

**Comment by jreback** [comment]
can you merge master and get this passing and will have a look

**Comment by jreback** [comment]
closing as stale, if you'd like continue, pls ping.

**Pull request closed without merging**